src/server/scripts/Events/fireworks_show.cpp

```diff
@@ -0,0 +1,903 @@
+/*
+* Copyright (C) 2008-2016 TrinityCore <http://www.trinitycore.org/>
```

Comment:
TC CodeStyle: add 1 space in front of line 2-16 just like the other source headers.